shared/src/main/scala/ListHelpers.scala

```diff
@@ -107,6 +130,14 @@ object ListHelpers:
 
   end map
 
+  def maximum(iterable: VList)(using ctx: Context): VAny =
+    iterable match
+      case VList() => VList()
+      case VList(list*) =>
+        list.reduce { (a, b) =>
+          if MiscHelpers.compareExact(a, b) > 0 then a else b
+        }
```

Comment:
Can you do something like
```
list.min(Ordering.fromLessThan[(VAny, VAny)](MiscHelpers.compareExact(_, _)))
```

Reply:
Maybe, but I prefer the style of the reduce-by better.

Reply:
Yes, but are you sure this isn't faster?